index.html

```diff
@@ -10,29 +10,32 @@
   <!-- header 시작 -->
   <header class ="header">
     <a href= "'/'">
-      <img src="../images/Property 1=sm.png" alt="판다마켓" class="header_img"></a>
-    <a href="login.html" target="_blank" class="login_btn">
-      <img src="../images/btn_small.png" alt="btn_small.png" class="login">
+      <img src="./images/Property 1=sm.png" alt="판다마켓" class="header_img">
+    </a>
+    <a href="login.html" target="_self" class="login_btn">
+      <img src="./images/btn_small.png" alt="btn_small.png" class="login">
```

Comment:
❗️ 수정요청
성오님 이 피드백은 2주차에도 드렸지만 중요한 내용이라 다시 남깁니다~
구경하러 가기 버튼을 바꿔주신 것처럼 로그인 버튼도 변경해주세요!

```suggestion
      로그인
```